### ISSUE TITLE
MultiSelect:  aria-label is not given to the div where type ="checkbox"

### ISSUE THREAD

**Issue by ramtejsudani6** [primefaces/primeng#12115]
### Describe the bug

When using MultiSelect component, we are running accessibility insights on the webpage and observed Aria-Label is not defined for checkbox at the multiselect-header.

<img width="354" alt="image" src="https://user-images.githubusercontent.com/83031111/199165335-ee8e2505-ad17-4614-b2c8-f8600e415b53.png">

The expected output is to have aria-label attribute to be defined for the checkboxes.

### Environment

Production

### Angular version

13.3.1

### PrimeNG version

13.3.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v14.18.3

### Steps to reproduce the behavior

We have verified the issue on below Primeng URL
URL: (https://www.primefaces.org/primeng/multiselect)

Steps to reproduce:
1. Expand the multiselect dropdown and open the inspect tool from browser.
2. Select the header checkbox element in the inspect tool and we find aria-label attribute is not available.

### Expected behavior

For Multiselect component, all the checkboxes should have aria-label attribute.

**Comment by ramtejsudani6** [comment]
This issue still exists in latest release

**Comment by msftedad** [comment]
Hi @ramtejsudani6 @cetincakiroglu , This issue still exists. Can you please reopen this issue. Thanks
